server/admin/Cargo.toml

```diff
@@ -13,3 +13,5 @@ lockbook-models = { path = "../../core/libs/models" }
 deadpool-redis = "0.10.1"
 redis = { version = "0.21", default-features = false, features = ["tokio-comp"] }
 redis_utils = "0.1.2"
+reqwest = "0.11.7"
+async-stripe = { git = "https://github.com/arlyon/async-stripe", default-features = true, features = ["runtime-tokio-hyper-rustls"] }
```

Comment:
we're waiting for https://github.com/arlyon/async-stripe/pull/170 to be merged before we can deploy ours right?

Reply:
Merged now :)